app/assets/javascripts/discourse/models/topic-tracking-state.js.es6

```diff
@@ -28,6 +29,10 @@ function isUnread(topic) {
   );
 }
 
+function isUnseen(topic) {
+  return !topic.last_seen_at || topic.last_seen_at < topic.created_at;
```

Comment:
When is `topic.last_seen_at` ever `< topic.created_at`?

Reply:
@ZogStriP thank you, very good point, I switched code to simply mark each post as read for a specific category.